docs/introduction-to-smart-contracts.rst

```diff
@@ -181,21 +181,19 @@ arguments ``from``, ``to`` and ``amount``, which makes it possible to track
 transactions.
 
 To listen for this event, you could use the following
-JavaScript code, which uses `web3.js <https://github.com/web3/web3.js/>`_ to create the ``Coin`` contract object,
+JavaScript code, which uses `ethers.js <https://github.com/ethers-io/ethers.js>`_ to create the ``coin`` contract object,
 and any user interface calls the automatically generated ``balances`` function from above:
 
 .. code-block:: javascript
 
-    Coin.Sent().watch({}, '', function(error, result) {
-        if (!error) {
-            console.log("Coin transfer: " + result.args.amount +
-                " coins were sent from " + result.args.from +
-                " to " + result.args.to + ".");
-            console.log("Balances now:\n" +
-                "Sender: " + Coin.balances.call(result.args.from) +
-                "Receiver: " + Coin.balances.call(result.args.to));
-        }
-    })
+    coin.on('Sent', async (from, to, amount, event) => {
+      console.log(`Coin transfer: ${ethers.formatUnits(amount, 18)} coins were sent from ${from} to ${to}.`);
+    
+      const senderBalance = await contract.balances(from);
+      const receiverBalance = await contract.balances(to);
```

Comment:
```suggestion
      const senderBalance = await coin.balances(from);
      const receiverBalance = await coin.balances(to);
```

I gues this should be ``coin`` as that is the name used before to list to the event, right?

Reply:
> I gues this should be `coin` as that is the name used before to listen to the event, right?

Good catch, thanks!